docs/products/m3db/index.rst

```diff
@@ -33,12 +35,43 @@ Check out our :doc:`getting-started` guide for Aiven for M3 and take your first
     💻 :doc:`howto`
 
 
-M3DB resources
---------------
+Ways to use Aiven for M3
+------------------------
+Handle and analyse the time-stamped data from multiple connected devices and services, scale up as needed, and compare datasets to provide insights into past and present.
+
+With Aiven for M3, you can set up the following example solutions:
+
+- Monitor IoT deployments, and use time series analysis to identify seasonal patterns, average usage and potential inefficiencies.
+
+- Measure the performance of your applications, detect problems, respond to incidents and plan maintenance. 
+
+- Cross-reference contextual data to understand financial trends. 
+
+- Process self-driving car data with faster data ingest and queries, and with stronger data compression. 
+
+- Predict future sales by using statistical models with historic data and cross-referencing it with consumer trends. 
+
+
+
+Works best with other Aiven products
```

Comment:
Grafana is the only Aiven product, we don't offer Prometheus - and we do offer InfluxDB but M3 replaces it, you wouldn't probably use them together. How about calling this section "Plays nicely with others" or "Integrates with your existing tools" or something like that?

Reply:
Good suggestion, I will reword.